SpatialGDK/Source/Public/Schema/Rotation.h

```diff
@@ -69,9 +69,18 @@ struct Rotation : Component
 	void ApplyComponentUpdate(const Worker_ComponentUpdate& Update)
 	{
 		Schema_Object* ComponentObject = Schema_GetComponentUpdateFields(Update.schema_type);
-		Pitch = Schema_GetDouble(ComponentObject, 1);
-		Yaw = Schema_GetDouble(ComponentObject, 2);
-		Roll = Schema_GetDouble(ComponentObject, 3);
+		if (Schema_GetFloatCount(ComponentObject, 1) > 0)
```

Comment:
Shouldn't this be `Schema_GetFloatCount(ComponentObject, 1) == 1` ?

Reply:
Technically yes, although this code would still work if it were a repeated field. Will change.